### PR TITLE
Prepare for the MP 6.1 release

### PR DESCRIPTION
**Pull request by Emily-Jiang**

### CODE REVIEW

File: spec/src/main/asciidoc/architecture.adoc [hunk 1]
```diff
@@ -33,6 +33,43 @@ This specification documents the component specification contents of each releas
 The specifications for each release are listed in reverse chronological order.
 
 [[microprofile6.0]]
+=== MicroProfile 6.1-RC1
+
+MicroProfile 6.1 still brings in the Jakarta EE 10 Core Profile.
```
Comment (jclingan) [inline spec/src/main/asciidoc/architecture.adoc:38]: Please update to "MicroProfile 6.1 includes support for Jakarta EE Core Profile 10". Notice this also appends the version number at the end to reflect the name format on the Core Profile page (https://jakarta.ee/specifications/coreprofile/)

File: spec/src/main/asciidoc/architecture.adoc [hunk 2]
```diff
@@ -33,6 +33,43 @@ This specification documents the component specification contents of each releas
...
+
+Based on MicroProfile's time-boxed release process, this is a minor release that does not include backward incompatible changes. Like MicroProfile 6.0, this release requires Jakarta EE Core Profile 10.
+
+If you are still dependent on Jakarta EE 9.1, please consider using the https://github.com/eclipse/microprofile/releases/tag/5.0[5.0 release of MicroProfile].
+If you are still dependent on Jakarta EE 8, please consider using the https://github.com/eclipse/microprofile/releases/tag/4.1[4.1 release of MicroProfile].
+If you are still dependent on Java EE 8, please consider using the https://github.com/eclipse/microprofile/releases/tag/3.3[3.3 release of MicroProfile].
+If you are still dependent on Java EE 7, please consider using the https://github.com/eclipse/microprofile/releases/tag/1.4[1.4 release of MicroProfile].
```
Comment (jclingan) [inline spec/src/main/asciidoc/architecture.adoc:45]: We should chat about when to eliminate some of these comments. How far should we go back?

File: spec/src/main/asciidoc/architecture.adoc [hunk 3]
```diff
@@ -33,6 +33,43 @@ This specification documents the component specification contents of each releas
...
+
+The complete list of MicroProfile 6.0 specifications includes:
```
Comment (jclingan) [inline spec/src/main/asciidoc/architecture.adoc:47]: Update to "MicroProfile 6.1"

**Review by jclingan — CHANGES_REQUESTED** [review]
I left comments on the proposed changes.

File: spec/src/main/asciidoc/architecture.adoc (hunk 2 above)
Comment (Emily-Jiang) [reply spec/src/main/asciidoc/architecture.adoc:45]: I think the list is okay. Maybe we can keep just 4. When we move up to EE 11, we can drop Java EE 7.

File: spec/src/main/asciidoc/architecture.adoc (hunk 3 above)
Comment (Emily-Jiang) [reply spec/src/main/asciidoc/architecture.adoc]: done

File: spec/src/main/asciidoc/architecture.adoc (hunk 1 above)
Comment (Emily-Jiang) [reply spec/src/main/asciidoc/architecture.adoc]: done

**Comment by Emily-Jiang** [comment]
@jclingan I have addressed your comments. Please take a look to see whether you are happy with them.

**Pull request merged**